src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseHeaderLabel.cs

```diff
@@ -63,8 +99,20 @@ public override int GetHashCode()
             return LabelAsInt32.GetRandomizedHashCode();
         }
 
+        /// <summary>
+        /// Determines whether two specified instances of <see cref="CoseHeaderLabel"/> are equal.
```

Comment:
There aren't explicit docs for op_Equality, but the general rule in the wiki seems to be not to link in the type, but to use a "generic description" of the type.

```suggestion
        /// Determines whether two specified COSE header label instances are equal.
```

Reply:
just to be consistent with the rest of the documentation.
```suggestion
        /// Determines whether two specified header label instances are equal.
```